### DISCUSSION TITLE
Search Returning unwanted Results

### DISCUSSION THREAD

**Discussion by Phil89a** [krisk/Fuse#777] [Q&A] [answered]
As I am just getting going with Fuse, I am likely doing something wrong - so would appreciate some help.

With the code below, when I search for a specific value, I am getting both items return, even though the value is found in only one of the items...

Why is this happening ?

Thanks

```
var list = [
  {
    categories:	null,
    contents:	"Maentwrog - Snowdonia - North Wales\n",
    title:	"Autumn Oak",
  },
  {
    categories:	null,
    contents:	"Borrowdale - Lake District - England\n",
    title:	"Autumn Rains",  
  }
];

function mySearch() {
	var resultJSON = fuse.search("wales");
	console.log(JSON.stringify(resultJSON, null, 3));
}

var options = {
	keys: ["title", "contents","categories"]
};
var fuse = new Fuse(list, options); 

mySearch();
```

This produces output of....

```
"[
   {
      'item': {
         'categories': null,
         'contents': 'Maentwrog - Snowdonia - North Wales\n',
         'title': 'Autumn Oak'
      },
      'refIndex': 0
   },
   {
      'item': {
         'categories': null,
         'contents': 'Borrowdale - Lake District - England\n',
         'title': 'Autumn Rains'
      },
      'refIndex': 1
   }
]"
```

**Comment by krisk** [comment, marked as answer]
Late reply — sorry about that.

This is expected with the default settings. Fuse is a *fuzzy* search library, so "wales" will partially match other short strings like "Rains" (they share common characters). The default `threshold` of `0.6` is fairly permissive.

A couple of things you can do:

1. **Lower the threshold** — e.g. `threshold: 0.3` will require closer matches.
2. **Enable `includeScore`** — you'll see the second result has a much worse (higher) score, so you can filter or visually distinguish them.
3. **Use extended search** — if you want exact substring matching, `useExtendedSearch: true` with the `'wales` (include-match) operator will only return items containing that exact substring.
